src/commands/restart.civet

```diff
@@ -0,0 +1,23 @@
+{ SlashCommandBuilder, ChatInputCommandInteraction, CacheType } from discord.js
+{ restart } from ../index.civet
+config from ../config.civet
+
+data := new SlashCommandBuilder()
+		.setName "restart"
+		.setDescription "Restarts the bot."

```

Comment:
What is the purpose of a restart command? Is the idea that someone upgrades the bot itself (not just Civet) and then uses a command to trigger a restart? I'm not really sure this is useful...